### DISCUSSION TITLE
Vectors with additional data

### DISCUSSION THREAD

**Discussion by a-giorgi** [milvus-io/milvus#4873] [Q&A and General discussion] [answered]
Hi, I'm new to milvus and I'd like to use it for an important project but I have a question for you:
Is it possible to store within an entity some information (i.e. strings) which however is not used to compute similarity?
Let me better explain my question:
I have lots of object like this:
{signature_value, url}
that, for example, form the following dataset:
[{537892, "http://url1.com"}
 {563728, "http://url2.com"}
 ...
]
I want be able to create a collection within milvus server containing that dataset and, when I use the search() method, **use only the signature_value to compute the similarity**.
Then, using the get_entity_by_id I want to be able o retrieve the full entity containing both the signature_value and the url.
Is that possible? And If the answer is yes, could you please provide me some examples that I can follow? 
Thank you in advance!

**Comment by shengjun1985** [comment, marked as answer]
Thank you for your concern and support for Milvus.
The current version v1.x does not support the feature.
At present, you can only manage this information externally.
But the developing version v2.x will support it.
Then it will be able to meet your needs.

**Reply by a-giorgi** [reply]
Thank you very much for your answer!
